js/site.js

```diff
@@ -1,7 +1,73 @@
+/*
+ * Each Site creates its own Score instance. The attributes
+ * of the Score are updated as we process new events e.g. trackers
+ * blocked or https status.
+ *
+ * The Score attributes are then used generate a site
+ * privacy score used in the popup.
+ */
+const siteScores = ['A', 'B', 'C', 'D']
+
+class Score {
+    constructor() {
+        this.noHTTPS = false;
+        this.topCompany = false;
+        this.totalBlocked = 0;
+        this.obscureTracker = false;
```

Comment:
I suggest these naming changes

- noHTTPS -> hasHTTPS
- topCompany -> inMajorTrackingNetwork
- objscureTracker -> hasObscureTracker

I know this would flip the HTTPS logic, I just find noHTTPS to feel odd.